src/IECoreHoudini/OBJ_SceneCacheTransform.cpp

```diff
@@ -31,7 +31,8 @@
 //  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 //
 //////////////////////////////////////////////////////////////////////////
-
+#include "UT/UT_Version.h"
+#include "OBJ/OBJ_Node.h"
```

Comment:
I'll remove OBJ/OBJ_Node.h . UT/UT_Version.h defines the UT_MAJOR_VERSION though.